docs/UserGuide.md

```diff
@@ -194,6 +218,30 @@ deletes the friend tag of the first person in the list.
 Disallowed examples:
 * `deltag 2 t/classmate t/neighbour` will not succeed as it tries to delete 2 tags at once.
 
+### Categorizing a tag : `cattag`
+
+Categorizes a tag under a defined category.
+
+Format: `cattag t/TAG [t/MORE_TAGS] CATEGORY`
+
+* Sets the tag(s) in CampusConnect with the specified name `TAG` to fall under the specified `CATEGORY`.
+* Currently available categories and their respective keywords are:
+
+    Category      | Keyword (case sensitive)
+    --------------|-------------------------
+    **General**   | `general`
+    **Academics** | `acads`
+    **Activities**| `activity`
+    **Networking**| `network`
+    **Mentorship**| `mentor`
+* Multiple tags can be categorized to the same category at a time.
```

Comment:
Add that `cattag` cannot categorize multiple tags into multiple categories, eg:
```
cattag t/tag1 acads t/tag2 t/tag3 general
```
is not allowed